darts/timeseries.py

```diff
@@ -880,6 +880,41 @@ def is_within_range(self,
         index = self.time_index()
         return index[0] <= ts <= index[-1]
 
+
+    def map(self,
+            fn: Callable[[float], float],
+            cols: Optional[Union[List[int], int]] = None) -> 'TimeSeries':
```

Comment:
@grll you'll probably have to change this as part of you refactoring

Reply:
yes we would need to replace `int` indexing  with `str` indexing  but nothing too serious here